rero_ils/modules/loans/api.py

```diff
@@ -290,6 +290,122 @@ def date_fields2str(self):
+                locations[location_pid] = LocationsSearch()\
+                    .filter('term', pid=location_pid)\
+                    .source(includes=['pid', 'name', 'library'])\
+                    .execute()[0].to_dict()
+            return locations[location_pid]
+
+        def library_name_by_pid(library_pid):
+            """Get library name by pid."""
+            if library_pid not in libraries:
+                libraries[library_pid] = LibrariesSearch()\
+                    .filter('term', pid=library_pid)\
+                    .source(includes='name')\
+                    .execute()[0].to_dict()['name']
+            return libraries[library_pid]
+
+        def item_by_pid(item_pid):
+            """Get item by pid."""
+            if item_pid not in items:
+                items[item_pid] = ItemsSearch()\
+                    .filter('term', pid=item_pid)\
+                    .filter('term', status=ItemStatus.ON_SHELF)\
+                    .source(includes=[
+                        'pid', 'barcode', 'call_number', 'library',
+                        'location', 'temporary_item_type'
+                    ]).execute()
+            return items[item_pid]
+
+        def item_type_by_pid(item_type_pid):
+            """Get item type by pid."""
+            if item_type_pid not in item_types:
+                item_types[item_type_pid] = ItemTypesSearch()\
+                    .filter('term', pid=item_type_pid)\
+                    .filter('term', negative_availability=True)\
+                    .execute()
+            return item_types[item_type_pid]
+
+        metadata = []
+        item_pids = []
+
+        loans = loans_pending()
+        for loan in loans:
+            item_pid = loan['item_pid']['value']
+            item = item_by_pid(item_pid)
+            if (len(item) > 0):
+                add = True
+                item = item[0]
+                if 'temporary_item_type' in item:
+                    item_type_pid = item['temporary_item_type']['pid']
+                    itemType = item_type_by_pid(item_type_pid)
+                    add = True if len(itemType) == 0 else False
+                if add and item_pid not in item_pids:
+                    item_pids.append(item_pid)
+                    item_data = item.to_dict()
+                    loan_data = loan.to_dict()
+                    item_data['library']['name'] = library_name_by_pid(
+                        item_data['library']['pid'])
+                    item_data['location']['name'] = location_by_pid(
+                        item_data['location']['pid'])['name']
+                    patron_data = patron_by_pid(loan_data['patron_pid'])
+                    patron_first_name = patron_data['first_name']
+                    patron_last_name = patron_data['last_name']
+                    loan_data['patron'] = {
+                        'barcode': patron_data['patron']['barcode'],
+                        'name': f'{patron_first_name} {patron_last_name}'
```

Comment:
You can directly do:
```suggestion
                        'name': f'{patron_data["first_name"]} {patron_data["last_name"]}'
```
Instead of storing in a variable